cpp/ql/src/jsf/4.10 Classes/AV Rule 77.1.ql

```diff
@@ -2,8 +2,8 @@
  * @name Constructor with default arguments will be used as a copy constructor
  * @description Constructors with default arguments should not be signature-compatible with a copy constructor when their default arguments are taken into account.
  * @kind problem
- * @problem.severity error
- * @precision high
+ * @problem.severity warning
+ * @precision low
```

Comment:
This is a pretty drastic change, but I agree this should be a `warning` and many of the results I found for this query don't look like they need addressing.  So :+1:.